lib/Library.php

```diff
@@ -182,7 +182,7 @@ public static function FilterArtwork(string $query = null, string $status = null
 			$statusCondition = 'true';
 		}
 		elseif($status == 'all-submitter' && $submitterUserId !== null){
-			$statusCondition = 'Status = ? or (Status = ? and SubmitterUserId = ?)';
+			$statusCondition = '(Status = ? or (Status = ? and SubmitterUserId = ?))';
```

Comment:
These parentheses aren't necessary, but I added them to be consistent with the line I added below (where they are necessary).